testing/tests/DevExpress.knockout/textBox.tests.js

```diff
@@ -0,0 +1,30 @@
+import ko from 'knockout';
+import TextBox from 'ui/text_box';
+
+import 'integration/knockout';
+
+const PLACEHOLDER_CLASS = 'dx-placeholder';
+
+const shouldRunTest = QUnit.urlParams['nojquery'] && QUnit.urlParams['nocsp'];
+
+if(shouldRunTest) {
+    QUnit.module('textBox');
+} else {
+    QUnit.module.skip('textBox');
+}
+
+QUnit.testStart(function() {
+    const markup = '<div id="text-box">';
+
+    document.getElementById('qunit-fixture').innerHTML = markup;
```

Comment:
```suggestion
    document.getElementById('qunit-fixture').innerHTML = markup;
```
const textBoxContainer = document.createElement('div');

textBoxContainer.setAttribute('id', 'text-box');
document.getElementById('qunit-fixture')
    .appendChild(textBoxContainer);

Reply:
direct applying mrkup to innerHTML can be not secure(not in this case, but let's do not use innerHTML when we can)

Reply:
No problem bro. I agree with you. It was fixed here: [[refactor(textBox: tests): Replace innerHTML with appendChild](https://github.com/DevExpress/DevExtreme/pull/24830/commits/22dc106651135dc13faaa3e1cfafc79680a5fe23)].